psi4/src/export_fock.cc

```diff
@@ -67,6 +67,9 @@ void export_fock(py::module &m) {
         .def("set_do_K", &JK::set_do_K)
         .def("set_do_wK", &JK::set_do_wK)
         .def("set_omega", &JK::set_omega)
+        .def("set_omega_alpha", &JK::set_omega_alpha)
+        .def("set_omega_beta", &JK::set_omega_beta)
```

Comment:
add a quick docstring and then an extra argument `"beta"_a`. this will mean that in python ppl can call `set_omega_beta(beta=0.4)`. that in itself is uncommon, but handy for docstring.